tests/h/search/index_test.py

```diff
@@ -93,6 +106,141 @@ def test_you_can_filter_annotations_by_creation_time(self, factories, index, sea
         response = search.filter("range", created={"gte": before}).execute()
         assert SearchResponseWithIDs([annotation.id]) == response
 
+    def test_you_can_filter_annotations_by_updated_time(self, factories, index, search):
+        update_time = datetime.datetime.now()
+        annotation = factories.Annotation.build(id="test_annotation_id", updated=update_time)
+
+        index(annotation)
+
+        response = search.filter("range", updated={"gte": update_time}).execute()
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_id(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_ann_id")
+
+        index(annotation)
+
+        response = search.filter("term", id="test_ann_id").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    @pytest.mark.parametrize('user_search_str', [
+        'acct:someone@example.com',
+        'someone',
+        'someone@example.com'
+    ])
+    def test_you_can_filter_annotations_by_user(self, factories, index, search, user_search_str):
+        annotation = factories.Annotation.build(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        response = search.filter("term", user=user_search_str).execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_make_aggregations_on_user_raw(self, factories, index, search):
+        annotation_1 = factories.Annotation.build(userid="acct:someone@example.com")
+        annotation_2 = factories.Annotation.build(userid="acct:Someone@example.com")
+
+        index(annotation_1)
```

Comment:
You can pass multiple annotations to `index` - `index(annotation_1, annotation_2)`. It is slightly more efficient to do this because each call to `index` triggers a "refresh" of the search index in order to make the changes visible in search results. Indexing several annotations in one call means that only one refresh operation is needed.